packages/tests/unit-tests/test-cases/expected/inferno-hooks/portal.tsx

```diff
@@ -1,5 +1,5 @@
-import { MutableRefObject } from 'react';
-import { createPortal } from 'react-dom';
+import { createPortal } from 'inferno';
```

Comment:
Rewrite it (and react generator) to use Portal component like it has done in inferno-generator